implementations/rust/ockam/ockam_command/src/util/config.rs

```diff
@@ -104,7 +104,8 @@ impl AuthoritiesConfig {
         let mut cfg = self.inner.write();
         cfg.add_authority(i, a);
         drop(cfg);
-        self.inner.persist_config_updates()
+        self.inner.persist_config_updates()?;
```

Comment:
I think that if you drop the `?;` then the last `Ok(())` is not necessary

Reply:
Not 100% sure, but I think I added that to make the compiler happy (due to error conversions). I'll review it.

Reply:
Yeah exactly, it would look like: `Ok(self.inner.persist_config_updates()?)`

Reply:
Just `self.inner.persist_config_updates()` doesn't compile? Isn't it returning the same `Result` type?

Reply:
No, that part of the code is old and it works with anyhow. I have plans to remove it altogether, that's why I didn't modify it.

Reply:
I get it, thanks.